packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -406,19 +412,38 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
 
   let filteredLatestErrors: ActiveErrorMessage[] = props.errorBarProps !== false ? props.latestErrors : [];
 
+  /* @conditional-compile-remove(notifications) */
+  let filteredLatestErrorNotifications: ActiveNotification[] = props.showErrorNotifications
+    ? (props.latestErrors as ActiveNotification[])
+    : [];
+
   const isCameraOn = useSelector(localVideoSelector).isAvailable;
 
   // TODO: move this logic to the error bar selector once role is plumbed from the headless SDK
-  if (role === 'Consumer' && props.errorBarProps) {
+  if (
+    role === 'Consumer' &&
```

Comment:
Do we know if this role information is coming from the SDK now and we can handle this TODO here?

Reply:
I don't think so but I can double check with Miguel